### DISCUSSION TITLE
Add custom icon to output exe (windows)

### DISCUSSION THREAD

**Discussion by KingKDot** [yao-pkg/pkg#15] [Ideas]
self explanitory

**Comment by robertsLando** [comment]
PR?

**Reply by KingKDot** [reply]
> PR?

I would love to but I'm not nearly good enough in js to help as much as I would like.

**Reply by robertsLando** [reply]
https://github.com/vercel/pkg/issues/1894

**Reply by damartripamungkas** [reply]
> https://github.com/vercel/pkg/issues/1894

it works if you open the .exe file but if you look at the file manager it remains as usual (GREEN BOX)

**Reply by robertsLando** [reply]
Try to check on vercel issues, there is someone who have been able to archieve that

**Reply by damartripamungkas** [reply]
> Try to check on vercel issues, there is someone who have been able to archieve that

I can't find it, if it's not a hassle can you help me? Thanks.

**Reply by robertsLando** [reply]
I don't have a reference right now I just remember there were many users asking for this and someone proposed solutions, never tried them as I'm not a windows user so I cannot help you so much
